packages/feedback-service/src/typedef.graphql

```diff
@@ -0,0 +1,52 @@
+type ProfileType {
+  kerberosID: String
+  name: String
+  email: String
+}
+
+type TimestampType {
+  createdAt: DateTime
+  createdBy: ProfileType
+  modifiedAt: DateTime
+  modifiedBy: ProfileType
+}
+
+type FeedbackType {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  spa: String
+  portalFeedback: Boolean
+  timestamp: TimestampType
+  title: String
+  feedbackType: String
+}
+
+input FeedbackInput {
+  _id: String
+  description: String
+  iid: String
+  experience: String
+  module: String
+  portalFeedback: Boolean
+  timestamp: TimestampInput
+  title: String
+  feedbackType: String
+}
+
+type Query {
+  # Query Definitions
+  listFeedback: [FeedbackType]
+  getFeedback(_id: String): FeedbackType
```

Comment:
Avoid using `_id` in any interface or user facing things. Just `id` is clear and easy to understand and use.

```suggestion
  getFeedback(id: String): FeedbackType
```